src/index.html

```diff
@@ -49,7 +49,7 @@
   <meta name="theme-color" content="#3f51b5">
 
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons&display=block" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Mono:300&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Mono:300,400&display=swap" rel="stylesheet">
```

Comment:
do we need both? is the 300 level still used anywhere?

Reply:
I doubt that there's any reason to use Mono 300 anywhere.